src/cli/generator-html.ts

```diff
@@ -333,23 +333,20 @@ function generateExpression(expression: Expression | SimpleExpression, ctx: Gene
         return expression.value
     }
     else if (isSymbolReference(expression)) {
-        let value = ''
-        ctx.argumentStack[0].forEach(function (el) {
-            if ((el as any).name === expression.symbol.ref?.name) {
-                value = (el as any).value
-            }
-        })
-        return value
+        const values = ctx.argumentStack[0];
+        const lastIndex = values
+            .reduce<number>((maxIndex, el, currentIndex) => (el as any).name === expression.symbol.ref?.name ? currentIndex : maxIndex, -1);
+        return lastIndex === -1 ? '' : values[lastIndex] as string;
     }
     else if (isOperation(expression)) {
         let result, left, right
         if (isStringExpression(expression.left) || typeof (generateExpression(expression.left, ctx)) === 'string') {
-            left = `'${generateExpression(expression.left, ctx)}'`
+            left = "'" + generateExpression(expression.left, ctx) + "'";
```

Comment:
This might be some thing of personal taste but there is also a performance penalty. I suggest to use string concatenation for very simple (single-line) strings.